src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -4338,8 +4338,11 @@ void ObjectMgr::LoadPlayerInfo()
                 if (sWorld->getIntConfig(CONFIG_EXPANSION) < EXPANSION_LEGION && class_ == CLASS_DEMON_HUNTER)
                     continue;
 
-                // fatal error if no level 1 data
-                if (!playerInfo->get()->levelInfo || playerInfo->get()->levelInfo[0].stats[0] == 0)
+                if (sWorld->getIntConfig(CONFIG_EXPANSION) < EXPANSION_DRAGONFLIGHT && class_ == CLASS_EVOKER)
+                    continue;
+
+                // fatal error if no level 1 data except evoker
+                if (!playerInfo->get()->levelInfo || playerInfo->get()->levelInfo[0].stats[0] == 0 && class_ != CLASS_EVOKER)
```

Comment:
Would be better to add an additional check so it does check player info ABOVE level 58

Reply:
or filling the db with fake stats for level < 58

Reply:
Nah, better to add a additional check for 58, otherwise db will be cluttered :D

Reply:
i disagree with both of you, just add level 1 data with fake stats for evoker, no need for adding data for 1-57 and also no need for a level 58 check

Reply:
done, u right